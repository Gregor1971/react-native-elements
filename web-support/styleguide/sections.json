[
  {
    "name": "Buttons",
    "components": "../../src/buttons/[A-Z]*.js"
  },
  {
    "name": "Cards",
    "components": "../../src/+(card|pricing)/[A-Z]*.js"
  },
  {
    "name": "Forms",
    "components": "../../src/form/[A-Z]*.js"
  },
  {
    "name": "Icons",
    "components": "../../src/+(icons|social)/[A-Z]*.js"
  },
  {
    "name": "Inputs",
    "components": "../../src/+(checkbox|rating|slider)/[A-Z]*.js"
  },
  {
    "name": "Lists",
    "components": "../../src/list/[A-Z]*.js"
  },
  {
    "name": "Tiles",
    "components": "../../src/tile/[A-Z]*.js"
  },
  {
    "name": "misc",
    "components":
      "../../src/+(avatar|badge|divider|header|text)/+(badge|[A-Z])*.js"
  }
]
