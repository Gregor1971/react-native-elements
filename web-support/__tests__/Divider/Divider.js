import parseSamples from 'enzyme-styleguidist-sample-parser';
import sections, { options } from '../../samples';

parseSamples({ Divider: sections.Divider }, options);
