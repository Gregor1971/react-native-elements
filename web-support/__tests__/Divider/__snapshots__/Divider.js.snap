// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Divider: Divider: [ prop: noProps ] enzyme-mount: snapshot 1`] = `
<Divider>
  <View
    style={
      Array [
        105,
        undefined,
      ]
    }
  >
    <div
      className="rn-alignItems-1oszu61 rn-backgroundColor-lq4czt rn-borderTopStyle-1efd50x rn-borderRightStyle-14skgim rn-borderBottomStyle-rull8r rn-borderLeftStyle-mm0ijv rn-borderTopWidth-13yce4e rn-borderRightWidth-fnigne rn-borderBottomWidth-ndvcnb rn-borderLeftWidth-gxnn5r rn-boxSizing-deolkf rn-display-6koalj rn-flexShrink-1pxmb3b rn-flexBasis-7vfszb rn-flexDirection-eqz5dr rn-height-109y4c4 rn-marginTop-1mnahxq rn-marginRight-61z16t rn-marginBottom-p1pxzi rn-marginLeft-11wrixw rn-minHeight-ifefl9 rn-minWidth-bcqeeo rn-paddingTop-wk8lta rn-paddingRight-9aemit rn-paddingBottom-1mdbw0j rn-paddingLeft-gy4na3 rn-position-bnwqim"
    />
  </View>
</Divider>
`;

exports[`Divider: Divider: [ prop: noProps ] enzyme-render: snapshot 1`] = `
<div
  class="rn-alignItems-1oszu61 rn-backgroundColor-lq4czt rn-borderTopStyle-1efd50x rn-borderRightStyle-14skgim rn-borderBottomStyle-rull8r rn-borderLeftStyle-mm0ijv rn-borderTopWidth-13yce4e rn-borderRightWidth-fnigne rn-borderBottomWidth-ndvcnb rn-borderLeftWidth-gxnn5r rn-boxSizing-deolkf rn-display-6koalj rn-flexShrink-1pxmb3b rn-flexBasis-7vfszb rn-flexDirection-eqz5dr rn-height-109y4c4 rn-marginTop-1mnahxq rn-marginRight-61z16t rn-marginBottom-p1pxzi rn-marginLeft-11wrixw rn-minHeight-ifefl9 rn-minWidth-bcqeeo rn-paddingTop-wk8lta rn-paddingRight-9aemit rn-paddingBottom-1mdbw0j rn-paddingLeft-gy4na3 rn-position-bnwqim"
/>
`;

exports[`Divider: Divider: [ prop: noProps ] enzyme-shallow: snapshot 1`] = `
<View
  style={
    Array [
      105,
      undefined,
    ]
  }
/>
`;

exports[`Divider: Divider: [ prop: style ] enzyme-mount: snapshot 1`] = `
<Divider
  style={
    Object {
      "backgroundColor": "#071",
      "height": 5,
    }
  }
>
  <View
    style={
      Array [
        105,
        Object {
          "backgroundColor": "#071",
          "height": 5,
        },
      ]
    }
  >
    <div
      className="rn-alignItems-1oszu61 rn-borderTopStyle-1efd50x rn-borderRightStyle-14skgim rn-borderBottomStyle-rull8r rn-borderLeftStyle-mm0ijv rn-borderTopWidth-13yce4e rn-borderRightWidth-fnigne rn-borderBottomWidth-ndvcnb rn-borderLeftWidth-gxnn5r rn-boxSizing-deolkf rn-display-6koalj rn-flexShrink-1pxmb3b rn-flexBasis-7vfszb rn-flexDirection-eqz5dr rn-height-4v7adb rn-marginTop-1mnahxq rn-marginRight-61z16t rn-marginBottom-p1pxzi rn-marginLeft-11wrixw rn-minHeight-ifefl9 rn-minWidth-bcqeeo rn-paddingTop-wk8lta rn-paddingRight-9aemit rn-paddingBottom-1mdbw0j rn-paddingLeft-gy4na3 rn-position-bnwqim"
      style={
        Object {
          "backgroundColor": "rgba(0,119,17,1)",
        }
      }
    />
  </View>
</Divider>
`;

exports[`Divider: Divider: [ prop: style ] enzyme-render: snapshot 1`] = `
<div
  class="rn-alignItems-1oszu61 rn-borderTopStyle-1efd50x rn-borderRightStyle-14skgim rn-borderBottomStyle-rull8r rn-borderLeftStyle-mm0ijv rn-borderTopWidth-13yce4e rn-borderRightWidth-fnigne rn-borderBottomWidth-ndvcnb rn-borderLeftWidth-gxnn5r rn-boxSizing-deolkf rn-display-6koalj rn-flexShrink-1pxmb3b rn-flexBasis-7vfszb rn-flexDirection-eqz5dr rn-height-4v7adb rn-marginTop-1mnahxq rn-marginRight-61z16t rn-marginBottom-p1pxzi rn-marginLeft-11wrixw rn-minHeight-ifefl9 rn-minWidth-bcqeeo rn-paddingTop-wk8lta rn-paddingRight-9aemit rn-paddingBottom-1mdbw0j rn-paddingLeft-gy4na3 rn-position-bnwqim"
  style="background-color:rgba(0,119,17,1);"
/>
`;

exports[`Divider: Divider: [ prop: style ] enzyme-shallow: snapshot 1`] = `
<View
  style={
    Array [
      105,
      Object {
        "backgroundColor": "#071",
        "height": 5,
      },
    ]
  }
/>
`;
